<!DOCTYPE html>
<html>
    <head>
	<title> Inken Wohlers </title>
    <link rel="stylesheet" href="style.css">
<script type="text/javascript">
/*<![CDATA[*/
<!--
 function UnCryptMailto(s) {
	var n=0;
	var r="";
	for(var i=0; i < s.length; i++) {
		n=s.charCodeAt(i);
		if (n>=8364) {n = 128;}
		r += String.fromCharCode(n-(1));
	}
	return r;
 }
 function linkTo_UnCryptMailto(s)	{
	location.href=UnCryptMailto(s);
 }
// -->
/*]]>*/
</script>
    </head>
    <body>
    
<div align="center">
<table><tr><td width='760px'>
<p align="right" style="color:gray">Last updated on 13/07/2022</p>
<div align='top'>

<img src="img/header.png" width="100%">
<h1><font color="#e42032">I</font><font color="gray">nke</font><font color="#006aa3">n</font><font color="#95bc0e"> W</font><font color="gray">ohler</font><font color="#fabb00">s</font></i></h1>

<p>
<a href="https://www.systembiologie.uni-luebeck.de" target=”_blank”>Medical Systems Biology</a> <br/>
<a href="https://www.uni-luebeck.de" target=”_blank”>University of Lübeck</a> <br/>
Lübeck Institute of Experimental Dermatology <br/>
Marie-Curie-Straße <br/>
23562 Lübeck <br/>
Building 67 (BMF) <br/>
Email: <a href="javascript:linkTo_UnCryptMailto('nbjmup;Jolfo/XpimfstAvoj.mvfcfdl/ef');" title="Open window to send e-mail" class="mail">Inken.Wohlers(at)uni-luebeck(dot).de</a> <br/>
Phone:	+49 (0) 451 3101 8476 <br/>
Fax:	+49 (0) 451 3101 8404 <br/>
<a href="https://scholar.google.co.uk/citations?user=_1RAtmYAAAAJ&hl=en" target=”_blank”><img width="30px" src="img/scholar.png" /></a>
&nbsp;
<a href="https://www.researchgate.net/profile/Inken-Wohlers-2" target=”_blank”><img width="30px" src="img/researchgate.jpg" /></a>
&nbsp;
<a href="https://twitter.com/InkenWohlers" target=”_blank”><img width="30px" src="img/twitter.png" /></a>
</p>

<h2><font color="#e42032">N</font><font color="gray">ews</font></h2>
<p> 
Participate in an article collection I am co-editing: <a href="https://t.co/dQC8HMk366?amp=1" target=”_blank”>Long read sequencing - pitfalls, benefits and success stories</a>
</p>


<h2><font color="#006aa3">E</font><font color="gray">xpertise</font></h2>
<p>
<ul>
<li> Sequencing Data Analysis </li>
<li> Genomics </li> 
<li> Transcriptomics </li> 
<li> Bioinformatics </li> 
<li> Algorithms </li>
<li> Data Science </li> 
<li> Systems Medicine </li>
</ul>
</p>

<h2><font color="#95bc0e">B</font><font color="gray">iography</font></h2>

<p>
<table>
<tr><td>
2018 – current 
</td>
<td>
<a href="https://www.systembiologie.uni-luebeck.de" target=”_blank”>Medical Systems Biology</a>, <a href="https://www.uni-luebeck.de" target=”_blank”>University of Lübeck</a> / <a href="https://www.uksh.de" target=”_blank”>UKSH</a>
</td></tr>
<tr><td>
2015 – 2018 
</td>
<td>
LIGA, <a href="https://www.uni-luebeck.de" target=”_blank”>University of Lübeck</a>; funding by <a href="https://engelhorn-stiftung.de/" target=”_blank”>Peter and Traudel Engelhorn Foundation</a>
</td></tr>
<tr><td>
2012 – 2015
</td>
<td>
Genome Informatics, University of <a href="https://www.uni-due.de" target=”_blank”>Duisburg-Essen</a> / <a href="https://www.uk-essen.de/en/" target=”_blank”>Universitätsklinikum Essen</a>
</td></tr>
<tr><td>
2012 
</td>
<td>
PhD in Algorithmic Bioinformatics at <a href="https://vu.nl/en" target=”_blank”>VU Amsterdam</a>  
</td></tr>
<tr><td>
2008 – 2012
</td>
<td>
PhD work conducted at <a href="https://www.cwi.nl/" target=”_blank”>CWI</a> Amsterdam with scientific stays at <a href="https://www.inria.fr/en/centre-inria-rennes-bretagne-atlantique" target=”_blank”>INRIA Rennes</a> 
</td></tr>
<tr><td>
2006 – 2008 
</td>
<td>
MSc in Bioinformatics at <a href="https://www.fu-berlin.de/" target=”_blank”>FU Berlin</a> 
</td></tr>
<tr><td>
2005 – 2008
</td>
<td>
Researcher at <a href="https://www.charite.de/" target=”_blank”>Charité University Medicine Berlin</a>
</td></tr>
<tr><td>
2002 – 2005 
</td>
<td>
BSc in Computational Life Science at <a href="https://www.uni-luebeck.de" target=”_blank”>University of Lübeck</a>
</td></tr>
</table>
</p>

<h2><font color="#fabb00">C</font><font color="gray">urrent Projects</font></h2>

<p>
<ul>
    <li>Collaborations using the <a href="https://www.egyptian-genome.org/" target="_blank">Egyptian genome ressource</a> whose lead Bioinformatician I was </li>
	<li>ONT data analysis (with <a href="https://www.neurogenetics-luebeck.de/" target="_blank">Institute of Neurogenetics</a> and <a href="https://www.cardiogenetics-luebeck.de/" target="_blank">Institute for Cardiogenetics</a>)</li>
	<li>North-East African mitochondrial haplogroups (with <a href="https://www.lied.uni-luebeck.de/team-members/ibrahim-lab-genetics.html" target="_blank">Genetics Division</a> of LIED)</li>
	<li>Ancient introgression (with <a href="https://www.ikmb.uni-kiel.de/research/junior-research-groups/ancient-dna-research" target="_blank">Ancient DNA Lab</a> and <a href="https://www.ikmb.uni-kiel.de/research/genetics-bioinformatics" target="_blank">Genetics &amp; Bioinformatics</a> at CAU Kiel)</li>
	<li>Prediction of effects of synonymous disease variants (with <a href="https://www.cardiogenetics-luebeck.de/" target="_blank">Institute for Cardiogenetics</a>)</li>
</ul>
</p>

<h2><font color="#ec7404">P</font><font color="gray">ublications</font></h2>
<p>
    R. Saurabh, C. Fouodo, I. König, H. Busch and <u>I. Wohlers</u>. A survey of genome-wide association studies (GWAS), polygenic scores (PGS) and UK Biobank (UKB) highlights resources for autoimmune disease genetics. Accepted at <a href="https://www.frontiersin.org/articles/10.3389/fimmu.2022.972107/abstract" target=”_blank”>Front. Immunol.</a> 2022<br />
</p>
<p>
T. Lüth, J. La&szlig;, S. Schaake, <u>I. Wohlers</u>, J. Pozojevic, R. D. G. Jamora, R. L. Rosales, N. Brüggemann, G. Saranza, C. C. E. Diesta, K. Schlüter, R. 
Tse, C. J. Reyes, M. Brand, H. Busch, C. Klein, A. Westenberger and J. Trinh. Elucidating Hexanucleotide Repeat Number and Methylation within the X-Linked 
Dystonia-Parkinsonism (XDP)-Related SVA Retrotransposon in TAF1 with Nanopore Sequencing. <a href="https://www.mdpi.com/2073-4425/13/1/126" target="_blank">Genes</a>. 2022<br /> <p>
    <u>I.Wohlers</u>, H. Pott, S. Schaake, J. Trinh, H. Busch and K. Lohmann. Be aware of pitfalls: Bioinformatic analysis of Cas9-targeted Nanopore sequencing of the RFC1 repeat in CANVAS. <a href="https://www.mdsabstracts.org/abstract/be-aware-of-pitfalls-bioinformatic-analysis-of-cas9-targeted-nanopore-sequencing-of-the-rfc1-repeat-in-canvas/" target=”_blank”>Movement Disorders</a> 2021 <br/>
</p>
<p>
    M. Munz, M. Khodaygani, Z. Aherrahrou, H. Busch and <u>I. Wohlers</u>. <em>In silico</em> candidate variant and gene identification using inbred mouse strains. <a href="https://peerj.com/articles/11017/" target=”_blank”>PeerJ</a> 2021<br\>
</p>
<p>
    D. Prokopenko, S. L. Morgan, K. Mullin, O. Hofmann, B. Chapman, R. Kirchner, S. Amberkar, <u>I. Wohlers</u>, C. Lange, W. Hide, L. Bertram and R. E. Tanzi. Whole-genome sequencing reveals new Alzheimer’s disease-associated rare variants in loci related to synaptic function and neuronal development. <a href="https://alz-journals.onlinelibrary.wiley.com/doi/10.1002/alz.12319" target=”_blank”>Alzheimers. Dement.</a> 2021<br />
</p>
<p>
    M. Ohms, C. Ferreira, H. Busch, <u>I. Wohlers</u>, A. C. Guerra de Souza, R. Silvestre and T. Laskay. Enhanced glycolysis is required for antileishmanial functions of neutrophils upon infection with Leishmania donovani. <a href="https://www.frontiersin.org/articles/10.3389/fimmu.2021.632512/full" target=”_blank”>Front. Immunol.</a> 2021<br />
</p>
<p>
    M. Munz, <u>I. Wohlers</u>, E. Simon, T. Reinberger, H. Busch, A. S. Schaefer, J. Erdmann. Qtlizer: comprehensive QTL annotation of GWAS results. <a href="https://www.nature.com/articles/s41598-020-75770-7" target=”_blank”>Sci. Rep.</a> 2020<br />
</p>
<p>
    <u>I. Wohlers</u>, A. Kuenstner, M. Munz, M. Olbrich, A. Faehnrich, V. Calonga-Solís, C. Ma, M. Hirose, S. El-Mosallamy, M. Salama, H. Busch and S. Ibrahim. A personal and population-based Egyptian genome reference. <a href="https://www.nature.com/articles/s41467-020-17964-1" target=”_blank”>Nat. Commun.</a> 2020<br />
</p>
<p>
    S. T. Jünger, F. Andreiuolo, M. Mynarek, <u>I. Wohlers</u>, S. Rahmann, L. Klein-Hitpass, E. Dörner, A. Zur Mühlen, N. Velez-Char, K. von Hoff, M. Warmuth-Metz, R. D. Kortmann, B. Timmermann, A. von Bueren, S. Rutkowski, T. Pietsch. CDKN2A deletion in supratentorial ependymoma with RELA alteration indicates a dismal prognosis: a retrospective analysis of the HIT ependymoma trial cohort. <a href="https://link.springer.com/article/10.1007/s00401-020-02169-z" target=”_blank”>Acta Neuropathol</a>. 2020<br />
</p>
<p>
    S. T. Jünger, M. Mynarek, <u>I. Wohlers</u>, E. Dörner, A. zur Mühlen, N. Velez-Char, K. von Hoff, S. Rutkowski, M. Warmuth-Metz, R.-D. Kortmann, B. Timmermann, S. Rahmann, L. Klein-Hitpass, A. von Bueren, T. Pietsch. Improved risk-stratification for posterior fossa ependymoma of childhood considering clinical, histological and genetic features – a retrospective analysis of the HIT ependymoma trial cohort. <a href="https://actaneurocomms.biomedcentral.com/articles/10.1186/s40478-019-0820-5" target=”_blank”>Acta Neuropathol. Commun</a>.&nbsp; 2019<br />
</p>
<p>
    <u>I. Wohlers</u>, C. Schulz, F. Kilpert, L. Bertram. Alzheimer's disease risk SNPs show no strong effect on miRNA expression in human lymphoblastoid cell lines. <a href="https://www.sciencedirect.com/science/article/abs/pii/S019745801930291X" target=”_blank”>Neurobiol. Aging</a>. 2019 <br/>
</p>
<p>
    P. Takousis*, A. Sadlon*, J. Schulz, <u>I. Wohlers</u>, V. Dobricic, L. Middleton, C. M. Lill, R. Perneczky, L. Bertram. (* joint first authors) Differential expression of microRNAs in Alzheimer's disease brain, blood and cerebrospinal fluid: a systematic review and meta-analysis. <a href="https://alz-journals.onlinelibrary.wiley.com/doi/full/10.1016/j.jalz.2019.06.4952" target=”_blank”>Alzheimer's &amp; Dementia</a>. 2019<br />
</p>
<p>
    J. Schulz*, P. Takousis*, <u>I. Wohlers</u>, I. O. G. Itua, V. Dobricic, G. Rücker, H. Binder, L. Middleton, J. P. A. Ioannidis, R. Perneczky, L. Bertram, C. M. Lill. (* joint first authors) Meta-analyses identify differentially expressed microRNAs in Parkinson's disease. <a href="https://onlinelibrary.wiley.com/doi/full/10.1002/ana.25490" target=”_blank”>Ann Neurol</a>. 2019<br />
</p>
<p>
    <u>I. Wohlers</u> and L. Bertram. Taking genomics research to the next level: The Genotype-Tissue expression project. <a href="https://onlinelibrary.wiley.com/doi/abs/10.1002/mds.27445" target=”_blank”>Mov Disord</a>. 2018<br />
</p>
<p>
    <u>I. Wohlers</u>, L. Bertram and C. M. Lill. Evidence for a potential role of miR-1908-5p and miR-3614-5p in autoimmune disease risk using integrative bioinformatics.<a href="https://www.sciencedirect.com/science/article/pii/S0896841118301197?via%3Dihub" target=”_blank”> J Autoimmun</a>. 2018<br />
</p>
<p>
    B. Grüning* , R. Dale*, [...] <u>I. Wohlers</u> [...] and J. Köster. (* joint first authors) Bioconda: A sustainable and comprehensive software distribution for the life science. <a href="https://www.nature.com/articles/s41592-018-0046-7" target=”_blank”> Nat Methods</a>. 2018<br />
</p>
<p>
    P. Johansson, A. Bergmann, S. Rahmann, <u>I. Wohlers</u>, R. Scholtysik, M. Przekopowitz, M. Seifert, G. Tschurtschenthaler, G. Webersinke, U. Jäger, R. Siebert, L. Klein-Hitpass, U. Dührsen, J. Dürig and R. Küppers. Recurrent alterations of TNFAIP3 (A20) in T-cell large granular lymphocytic leukemia. <a href="https://onlinelibrary.wiley.com/doi/abs/10.1002/ijc.29697" target=”_blank”>Int J Cancer.</a> 2016<br />
</p>
<p>
    R. Andonov, H. Djidjev, G. W. Klau, M. Le Boudic-Jamin and <u>I. Wohlers</u>. Automatic Classification of Protein Structure Using the Maximum Contact Map Overlap Metric. Algorithms, <a href="http://www.mdpi.com/1999-4893/8/4/850" target=”_blank”>Special issue Algorithmic Themes in Bioinformatics.</a> 2015<br />
</p>
<p>
    C. M. Lill, F. Luessi, A. Alcina, E. A. Sokolova, N. Ugidos, B. de la Hera, L. Guillot-Noël, S. Malhotra, E. Reinthaler, B.-M. M. Schjeide, J. Y. Mescheriakova, A. Mashychev, <u>I. Wohlers</u> et al. Genome-wide significant association with seven novel multiple sclerosis risk loci.<a href="https://jmg.bmj.com/content/52/12/848.long" target=”_blank”> J Med Genet</a>. 2015<br />
</p>
<p>
    T. Pietsch, <u>I. Wohlers</u>, T. Goschzik, V. Dreschmann, D. Denkhaus, E. Dörner, S. Rahmann, and L. Klein-Hitpass. Supratentorial ependymomas of childhood carry C11orf95–RELA fusions leading to pathological activation of the NF-κB signaling pathway.<a href="https://link.springer.com/article/10.1007%2Fs00401-014-1264-4" target=”_blank”> Acta Neuropathol</a>. 2014<br />
</p>
<p>
    M. El-Kebir*, T. Marschall*, <u>I. Wohlers</u>*, M. Patterson, J. Heringa, A. Schönhuth, and G. W. Klau. (* joint first authors) Mapping proteins in the presence of paralogs using units of coevolution.<a href="https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-14-S15-S18" target=”_blank”> BMC Bioinformatics</a>. 2013<br />
</p>
<p>
    <u>I. Wohlers</u>, R. Andonov and G. W. Klau. DALIX: Optimal DALI protein structure alignment.<a href="https://ieeexplore.ieee.org/document/6365174/" target=”_blank”> IEEE/ACM Trans Comput Biol Bioinform</a>. 2013<br />
</p>
<p>
    <u>I. Wohlers</u>, N. Malod-Dognin, R. Andonov, and G. W. Klau. CSA: comprehensive comparison of pairwise protein structure alignments. <a href="https://academic.oup.com/nar/article/40/W1/W303/1071161" target=”_blank”>Nucleic Acids Res</a>. 2012<br />
</p>
<p>
    <u>I. Wohlers</u>, R. Andonov, and G. W. Klau. Algorithm engineering for optimal alignment of protein structure distance matrices. <a href="https://link.springer.com/article/10.1007/s11590-011-0313-3" target=”_blank”>Optimization Letters</a>. 2011<br />
</p>
<p>
    <u>I. Wohlers</u>, F. S. Domingues, and G. W. Klau. Towards optimal alignment of protein structure distance matrices. <a href="https://academic.oup.com/bioinformatics/article/26/18/2273/208531" target=”_blank”>Bioinformatics</a>. 2010<br />
</p>
<p>
    <u>I. Wohlers</u>*, H. Stachelscheid*, J. Borstlap, K. Zeilinger, and J. C. Gerlach. (* joint first authors) The Characterization Tool: A knowledge-based stem cell, differentiated cell, and tissue database with a web-based analysis front-end. <a href="https://www.sciencedirect.com/science/article/pii/S1873506109000567" target=”_blank”>Stem Cell Res</a>. 2009<br/>
</p>

<h2><font color="#e42032">P</font><font color="gray">reprints</font></h2>

<p>
    M. Thomsen*, A. Kuenstner*, <u>I. Wohlers</u>, M. Olbrich, T. Lenfers, T. Osumi, Y. Shimazaki, K. Nishifuji, S. M. Ibrahim, A. Watson<sup>§</sup>, H. Busch<sup>§</sup> and M. Hirose. (*joint first authors; <sup>§</sup>equal contribution). A comprehensive analysis of the gut and skin microbiota in canine atopic dermatitis in Shiba Inu dogs. <a href="https://www.biorxiv.org/content/10.1101/2022.07.11.497949v1" target=”_blank”>bioRxiv 2022.07.11.497949 </a>. 2022<br />
</p>
<p>
    A. Fähnrich*, I. Stephan*, M. Hirose, M. Ali Awadelkareem, S. Ibrahim, H. Busch<sup>§</sup> and <u>I. Wohlers</u><sup>§</sup>. (*joint first authors; <sup>§</sup>joint last authors). North and East African mitochondrial genetic variation needs further characterization towards precision medicine. <a href="https://www.biorxiv.org/content/10.1101/2021.12.10.472079v1" target=”_blank”>bioRxiv 2021.12.10.472079</a>. 2021 <br />
</p>

<p>
    <u>I. Wohlers</u>, V. Calonga-Solís, J.-N. Jobst and H. Busch. COVID-19 genetic risk and Neanderthals: A case study highlighting the importance of scrutinizing diversity. <a href="https://www.biorxiv.org/content/10.1101/2020.11.02.365551v2" target=”_blank”>bioRxiv 2020.11.02.365551</a>. 2020<br />
</p>

<h2><font color="#006aa3">B</font><font color="gray">achelor's, Master's and PhD thesis</font></h2>
<p>
Exact algorithms for pairwise protein structure alignment. <a href="https://research.vu.nl/files/42214900/complete%20dissertation.pdf" target=”_blank”>PhD thesis</a> at the Free University Amsterdam, December 2012. 
</p>
<p>
Alignment of protein distance matrices with mathematical programming. M.Sc. thesis at the Free University Berlin, August 2008. 
</p>
<p>
Direkte Suchmethoden und ihre Anwendung zur Schätzung endokriner Parameter. B.Sc. thesis at the University of Lübeck, August 2005.
</p>


</div>
</td>

<td width='200px'>
<div height='700px' align='top'>
<a class="twitter-timeline" href="https://twitter.com/InkenWohlers?ref_src=twsrc%5Etfw" data-tweet-limit="8">Tweets by Inken Wohlers</a> <script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script> 
</div>
</td></tr></table>

</div>

    </body>
</html>

